src/app/code/community/Shopgate/Cloudapi/etc/api2.xml

```diff
@@ -258,7 +259,14 @@
                         <route>/v2/customers/:customer_id/addresses</route>
                         <action_type>collection</action_type>
                     </route_v2_customers_id_addresses>
+                    <route_v2_customers_id_addresses_id>
+                        <route>/v2/customers/:customer_id/addresses/:id</route>
+                        <action_type>entity</action_type>
+                    </route_v2_customers_id_addresses_id>
                 </routes>
+                <attributes translate="ids" module="api2">
```

Comment:
there is already an attributes node above (maybe did not notice in merge?), it needs to be moved up.